src/Engine/Segments/SegmentConditionModel.php

```diff
@@ -154,4 +156,22 @@ private function matchesSemverTraitValue($trait, $value, $condition)
 
         return $condition;
     }
+
+    private function matchesModuloTraitValue(mixed $traitValue)
+    {
+        $valueParts = explode('|', $this->value);
```

Comment:
`explode` wow, PHP knows how to name things :laughing:

Reply:
Haha, yeah! There is also `implode()` instead of `reduce()`. Someone is having fun naming things in PHP towers.